docs/Queries.md

```diff
@@ -24,6 +24,17 @@ title: Queries
 
```

Comment:
can you rebase to master? `docs` were moved to `website/docs`

Reply:
Done :-)